pkg/resources/marketplace/implicit_catalog_source_reconciler.go

```diff
@@ -37,7 +40,7 @@ func NewImplicitCatalogSourceReconciler(log logger.Logger, client k8sclient.Clie
 // Reconcile finds the CatalogSource that provides the current installation,
 // returning an error if it fails to find it. Caches the found CatalogSource
 // to be used by r.CatalogSourceName() and r.CatalogSourceNamespace()
-func (r *ImplicitCatalogSourceReconciler) Reconcile(ctx context.Context) (reconcile.Result, error) {
+func (r *ImplicitCatalogSourceReconciler) Reconcile(ctx context.Context, subName string) (reconcile.Result, error) {
```

Comment:
@briangallagher I haven't code reviewed but with the addition of some logic to this function do you think it would be valuable to have a unit test for it now?

Reply:
@laurafitzgerald Unit test added 👍